### DISCUSSION TITLE
mfront - How to write functions in mfront?

### DISCUSSION THREAD

**Discussion by kkolari** [thelfer/tfel#463] [General]
I have tried to find an example to figure out how to write simple functions in mfront.
Is there a way to do it and how? (I am more familiar with Fortran than C/C++)
Thanks for advance

**Comment by thelfer** [comment]
@kkolari, do you mean material properties ? If so, you may have a look at this tutorial: https://thelfer.github.io/tfel/web/material-properties.html

**Reply by kkolari** [reply]
Thanks for a prompt reply - again! 
I'll check the link. I am writing a material damage model.

**Reply by thelfer** [reply]
Ok, this might have little to do with material properties then.
You may want to have a look at the MFront Gallery <https://thelfer.github.io/tfel/web/gallery.html> 
Among the few examples of the Damage section, you may consider reading this one: <https://thelfer.github.io/tfel/web/FichantLaBorderieDamageBehaviour.html>

**Reply by kkolari** [reply]
Thanks @thelfer! 
I'll try the examples in the link: "https://thelfer.github.io/tfel/web/FichantLaBorderieDamageBehaviour.html".
